portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +32,23 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
+/** Fetches messages from the server and adds them to the DOM. */
```

Comment:
[nit (short for nitpicky comment)] Google's big on following the Google programming language style guides when coding. These can sometimes be pretty subjective, but it is nice for everyone to use the same styling when you're working on lots of code together. Here's a couple of notes from the JavaScript one.

Put your "{" on the same line as your function definition:
https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks

Use "/* comment */" or "// comment" for single-line comments instead of "/** comment */":
https://google.github.io/styleguide/jsguide.html#formatting-comments